intl.emu

```diff
@@ -215,11 +201,12 @@
         1. <ins>If _unit_ is not *undefined* and IsWellFormedUnitIdentifier(_unit_) is *false*, throw a *RangeError* exception.</ins>
         1. Let _unitDisplay_ be ? GetOption(_options_, *"unitDisplay"*, ~string~, « *"short"*, *"narrow"*, *"long"* », *"short"*).
         1. If _style_ is *"currency"*, then
+          1. <ins>If _currency_ is *undefined*, throw a *RangeError* exception.</ins>
           1. Set _intlObj_.[[Currency]] to the ASCII-uppercase of _currency_.
```

Comment:
If _currency_ is undefined, we should not throw, but just not set _intlObj_.[[Currency]], as is done with _unit_ below.